src/data/events.ts

```diff
@@ -96,24 +163,28 @@ export async function getNext3Events(): Promise<[Event, Event, Event]> {
     date: new Date(event.start),
     description: event.shortDescription,
     link: event.slug ? `/events/${event.slug}` : null,
-    image: {
-      src: await getEventImage(event),
-    },
-  })) as [Promise<Event>, Promise<Event>, Promise<Event>];
+    image: await getEventImage(event.calendarCover.value),
+  })) as [Promise<EventCalendarEntry>, Promise<EventCalendarEntry>, Promise<EventCalendarEntry>];
   const result = await Promise.all(promises);
   return result;
 }
 
-async function getEventImage(event: EventCMS): Promise<GetImageResult> {
-  if (typeof event.calendarCover.value === 'string') {
-    return await getImage({ src: calendarCoverImage, width: 400, height: 400 });
+export async function getEventImage(image: string | Media | undefined): Promise<YVImage> {
+  if (!image || typeof image === 'string') {
```

Comment:
I think we should just not return an image if there is none, instead of this random picture.

Reply:
Yeah, for example for the timetable it may be ok to be null, but what about the calendarCover?

Reply:
If it's required in payload it shouldn't be empty, right? And if the type is `string` we should just throw an error here so the build fails and we get notified.